workspace-manager-clienttests/src/main/java/scripts/uploadscripts/SummariesToBigQuery.java

```diff
@@ -0,0 +1,215 @@
+package scripts.uploadscripts;
+
+import bio.terra.testrunner.collector.MeasurementCollectionScript;
+import bio.terra.testrunner.collector.MeasurementCollector;
+import bio.terra.testrunner.common.utils.BigQueryUtils;
+import bio.terra.testrunner.runner.TestRunner;
+import bio.terra.testrunner.runner.TestScriptResult;
+import bio.terra.testrunner.runner.config.ServiceAccountSpecification;
+import bio.terra.testrunner.runner.config.TestConfiguration;
+import bio.terra.testrunner.runner.config.TestScriptSpecification;
+import bio.terra.testrunner.uploader.UploadScript;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.InsertAllRequest;
+import com.google.cloud.bigquery.TableId;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.nio.file.Path;
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.TimeZone;
+
+public class SummariesToBigQuery extends UploadScript {
+    private static final Logger logger = LoggerFactory.getLogger(SummariesToBigQuery.class);
+
+    /** Public constructor so that this class can be instantiated via reflection. */
+    public SummariesToBigQuery() {}
+
+    protected String projectId; // google project id
+    protected String datasetName; // big query dataset name
+
+    protected TestConfiguration renderedTestConfiguration;
+    protected TestRunner.TestRunSummary testRunSummary;
+    protected MeasurementCollectionScript.MeasurementResultSummary[] measurementCollectionSummaries;
```

Comment:
do these three need to be class-level variables? Could you make these variables local to `uploadResults()`, and then pass arguments as necessary to `buildMeasurementCollectionRow`, `buildTestScriptResultsRow`, `buildTestRunRow`?

Reply:
Good suggestion! I have recorded these in the enhancement [doc](https://docs.google.com/document/d/1gYqlJTpC4C4S1K0pYikeNJx-HWNZAZJYybinmuqcSTI/edit#). Will take a look at the needs of these variables and organize them accordingly.